foundationdb/src/tuple/single.rs

```diff
@@ -224,7 +228,7 @@ impl Decode for Uuid {
         let mut uuid = [0u8; 16];
         uuid.copy_from_slice(&buf[1..17]);
 
-        Ok((Uuid(uuid), 17))
+        Ok((Uuid::from_random_bytes(uuid), 17))
```

Comment:
We should use `from_bytes` because `from_random_bytes` forces random uuid variant.

https://docs.rs/uuid/0.6.3/src/uuid/lib.rs.html#743-748

Reply:
Silly me :) Let me update. Thanks!